raiden-dapp/CHANGELOG.md

```diff
@@ -8,10 +8,12 @@
 
 ### Fixed
 
+- [#1300] General screen shows Raiden account balance when using subkey.
```

Comment:
Wait why is this still unreleased?

Reply:
I think the change logs haven't been updated on v0.6.0 release

Reply:
Yes I need to rebase give me a moment

Reply:
Ah ok thank you 🙏